caffe2/python/data_parallel_model.py

```diff
@@ -205,6 +211,15 @@ def Parallelize_GPU(
         devices, model_helper_obj, model_helper_obj.param_init_net, sync_names
     )
 
+    # Handle any operations that need to be done after parameter sync
+    # i.e. making sure multi-precision copies of parameters are up-to-date
+    if post_sync_builder_fun is not None:
```

Comment:
No, I still need it. `optimizer_builder_fun` happens here: https://github.com/caffe2/caffe2/blob/master/caffe2/python/data_parallel_model.py#L183-L191 while `SyncParams` happens later, here: https://github.com/caffe2/caffe2/blob/master/caffe2/python/data_parallel_model.py#L222-L224

My sync needs to happen after the `SyncParams` call